src/state/reducers/myInfoReducer.js

```diff
@@ -0,0 +1,29 @@
+import {
+  FETCH_MY_INFO_START,
+  FETCH_MY_INFO_SUCCESS,
+  FETCH_MY_INFO_ERROR,
+} from '../actions/index';
+
+const initialState = {
+  seller_name: 'redux',
```

Comment:
It is better to use something like this:
const initialState = {
  myInfo: {},
  getMyInfoStatus: requestStatus.ready,
};
So we can  check if info is loading. As an example, look at addProductReducer.js .